playbook/app/pb_kits/playbook/pb_table/styles/_div_support.scss

```diff
@@ -0,0 +1,22 @@
+[class^=pb_table] {
+  .div-table {
```

Comment:
This is not needed if you [bring these on line 4](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_table/styles/_structure.scss#L4)